oak/server/dev/dev_oak_manager.cc

```diff
@@ -38,13 +38,43 @@ grpc::Status DevOakManager::CreateApplication(grpc::ServerContext* context,
   std::string application_id = NewApplicationId();
   LOG(INFO) << "Creating app with with id: " << application_id;
 
-  InitializeAssertionAuthorities();
-  runtime_->InitializeServer(
-      request->application_configuration(),
-      asylo::EnclaveServerCredentials(asylo::BidirectionalNullCredentialsOptions()));
+  auto runtime = absl::make_unique<OakRuntime>();
+  auto status = runtime->Initialize(request->application_configuration());
+  if (!status.ok()) {
+    return status.ToOtherStatus<grpc::Status>();
+  }
+
+  grpc::ServerBuilder builder;
```

Comment:
I thought we would have the server inside the runtime, rather than alongside it, or does that complicate things? Fine to do later in a separate PR of course.

Reply:
Well, that is the point of the whole refactor, to move the server to specific implementation vs in the runtime. The idea is that the server setup is very different for remote attestation vs current version and having the code this way allows to change the "asylo" version without affecting the common code.

Reply:
Sorry, this is the Oak Manager, not the Oak Runtime, my mistake. But actually, there is no reason for the Oak Manager to use enclave server credentials at all, that is and will always be an unattested and untrusted gRPC server, so standard TLS (or insecure credentials for dev) would be fine. Especially if this ends up being something similar to a Google Cloud API, in which case something like the `gcloud` tool needs to be able to interact with it. Apologies if I mixed Manager and Runtime in our discussions before, I probably did at some point!